recipe/meta.yaml

```diff
@@ -22,28 +21,36 @@ requirements:
     - python
     - pip
     - numpy
+    - setuptools
+    - wheel
   run:
     - python
     - {{ pin_compatible('numpy') }}
     - scipy >=0.9
 
 test:
+  requires:
+    - pip
   imports:
     - ecos
     - _ecos
   files:
     - test_interface_bb.py  # copied from repo; isn't in the sdist
   commands:
     - python test_interface_bb.py
+    - pip check
 
 about:
-  home: http://www.embotech.com/ECOS
-  license: GPL-3.0
+  home: https://github.com/embotech/ecos/wiki
+  description: 'This is the Python package for ECOS: Embedded Cone Solver. See Github page for more information.'
+  license: GPL-3.0-only
   license_family: GPL
   license_file: LICENSE  # in recipe for now; https://github.com/embotech/ecos-python/pull/15
-  summary: 'Python interface for ECOS, a lightweight conic solver for second-order cone programming'
+  summary: 'Python interface for ECOS a lightweight conic solver for second-order cone programming'
   dev_url: https://github.com/embotech/ecos-python
+  doc_url: https://github.com/embotech/ecos
```

Comment:
I am torn if the dev and doc url should be the same here as we are creating the python version.